.travis.yml

```diff
@@ -9,7 +9,8 @@ env:
 #  - HEADLESS="true"
 
 install:
-  - pip install -r requirements.txt
+  - pip3 install pipenv
```

Comment:
While user has mutiple version of `python3`, `pip3` might point to a package manager than the one we want. For exmaple, as I test on one the lab machine:

```
$ python3 --version
Python 3.7.3

$ which python3
/home/newslab/anaconda3/bin/python3

$ pip3 --version
pip 19.1.1 from /usr/local/lib/python3.5/dist-packages/pip (python 3.5)

$ python3 -m pip --version
pip 19.0.3 from /home/newslab/anaconda3/lib/python3.7/site-package/pip (python 3.7)
```

A better approach would be `python3 -m pip ...`, if `pip3` would just be used at `line 12`

Reply:
good call I'll make that change